project/FrontEnd/collaborative_science_platform/lib/screens/workspace_page/mobile_workspace_page/widget/contributor_card.dart

```diff
@@ -1,15 +1,19 @@
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:go_router/go_router.dart';
 import '../../../../models/user.dart';
+import '../../../../utils/colors.dart';
```

Comment:
can you update this styling. ıt should be similar to package:colloborative_science_platform/...